4.6.2/WcfBasicIISHost/WcfServiceTest/Web.config

```diff
@@ -29,6 +29,7 @@
 
   <system.web>
     <compilation debug="true" targetFramework="4.0" />
+    <customErrors mode="Off"/>
```

Comment:
I guess the customErrors may be turned off for debugging purpose, but I want to make sure it is intended you want to keep it off in the published sample.